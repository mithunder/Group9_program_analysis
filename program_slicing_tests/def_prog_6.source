module def_prog_6 :
a := 4;
do (true) -> skip
od;
@programslicing="true"
x := a
end
