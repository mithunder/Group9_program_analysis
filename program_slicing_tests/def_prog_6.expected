module def_prog_6 :
a := 4;
x := a
end
